content/components/github-shared-components.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: GitHub shared components
+description: Components that are shared by GitHub feature teams, but are not publicly available in Primer yet.
+---
+
+<Note>This information is only relevant to GitHub staff.</Note>
+
+These components support new design patterns that have been piloted by feature teams outside of Primer. While we encourage relying only on Primer components, we also value exploring new ideas without the exhaustive vetting process required for inclusion in Primer.
```

Comment:
```suggestion
These components are shared design patterns by GitHub feature teams developing application-specific components. We encourage relying only on Primer components where possible, and not all patterns will be upstreamed to Primer. 

```

Reply:
Hi @emilybrick and @mperrotti — minor edit to avoid confusion but I'd like to make sure that the:
1. Perception of our processes is respected (language to move away from exhausting/exhaustive toward more stable, trusted, resilient terms) 
2. Clear as day understanding that this is for GitHub application-level patterns that can, but may not be, upstreamed to Primer. It is not a draft state or experimental state. We are not "obscuring" any patterns or components from our Open Source library.

TLDR: can @mperrotti or @emilybrick run this through comparing against the language here? https://thehub.github.com/epd/engineering/dev-practicals/frontend/common-components/

Reply:
We were already pretty aligned with that post, but I just pushed up a few changes to make the alignment even stronger 💪